docs/source/tutorials/slowmo_ddp.rst

```diff
@@ -0,0 +1,86 @@
+Efficient Data Parallel Training with SlowMo Distributed Data Parallel
+======================================================================
+
+SlowMo Distributed Data Parallel reduces the communication between different
+nodes while performing data parallel training. Let's suppose that your trainer
```

Comment:
Does this look okay? Or, did you have other tradeoffs in mind?

Reply:
I believe it should at least mention that since the weights of the models on different nodes aren't kept in synch anymore after each iteration, the optimization dynamics are affected and this can lead to different results compared to DDP.

Reply:
Fixed this thanks

Reply:
Looks good, just `but Let's suppose that your trainer looks like` -> `Let's suppose that your trainer looks like`